sdk/pom.xml

```diff
@@ -261,6 +271,14 @@
     </repository>
   </repositories>
   <build>
+    <resources>
+      <resource>
+        <directory>${basedir}/../libevm/bin</directory>
```

Comment:
I think we can add custom step while building the shared libraries to put the result into the specific resource folder:
for example, linux: `resources/native/linux64/libevm.so`
windows: `resources/native/windows64/libevm.dll`
Similar to how act for `zendoo-sc-cryptolib`

Moreover: can we change the project structure to avoid including EVM JNA part to `sdk` module, but instead keeping everything go-library related inside another project.

So,  the root `.pom` file may consists of:
```
       <modules>
		<module>sdk</module> // depends on evm
		<module>libevm-go</module> // build go code only into 2 shared libraries (linux and windows) and copy them into evm module resourses
                <module>evm</module> // depends on libevm-go, build java classes with JNA and proper evm related java interfaces
		<module>examples/simpleapp</module>
		<module>tools/sctool</module>
	</modules>
```

Reply:
Regarding the resouces folder:
I added a copy-step to the build of `libevm` which puts the libraries into the `src/main/resources` folder of `evm`, which worked but had some side-effects: The libraries were not removed on `clean` and it is just an intermediate copy before they get copied to `evm/target`. Hence, I went back to the way it was done in the prototype: Directly reference the `libevm/bin` folder in `evm`, which will cause the libraries to be copied to the `evm/target` directory directly. I think it is cleaner this way.

Reply:
ok